apisix/plugins/sls-logger.lua

```diff
@@ -21,7 +21,7 @@ local bp_manager_mod = require("apisix.utils.batch-processor-manager")
 
 local plugin_name = "sls-logger"
 local ngx = ngx
-local rf5424 = require("apisix.plugins.slslog.rfc5424")
+local rf5424 = require("apisix.syslog.rfc5424")
```

Comment:
Please put the `syslog` library in the `plugins` folder. :)

Reply:
The sls-logger also use `rfc5424.lua`, Is there any confusion in put `rfc5424.lua` to `plugins/syslog` folder?

Reply:
@leslie-tsang @Sn0rt @monkeyDluffy6017 Hi, could give me some suggestion?

Reply:
The `apisix.syslog.rfc5424` is only used by plugin `sys-logger`, right?
So why not move the `apisix.syslog.rfc5424` to `apisix.plugins.slslog.rfc5424`?

Reply:
The `rfc5424.lua` both used by `syslog` and `slslogger`, so I moved the file to single other folder.

Reply:
> @leslie-tsang How about moving the file rfc5424 to `apisix/utils/` ?

LGTM

Reply:
Updated.